tests/Unit/TestingFramework.hpp

```diff
@@ -147,29 +145,44 @@ struct check_iterable_approx<
   }
 };
 
+template <typename T>
+struct check_iterable_approx<T, Requires<tt::is_a_v<std::unordered_set, T>>> {
```

Comment:
stick in ordered container and check that, if possible

Reply:
There appears to be no way to determine whether a type can be put into a set.  It would require SFINAE on function bodies, which doesn't exist.